prisma/migrations/20230226191138_add_redemption_kyc/migration.sql

```diff
@@ -0,0 +1,30 @@
+-- CreateEnum
+CREATE TYPE "KycStatus" AS ENUM ('NOT_STARTED', 'PASS', 'PENDING', 'FAIL_TRANSACTION', 'FAIL_MAX_ATTEMPTS');
+
+-- CreateEnum
+CREATE TYPE "DedupeStatus" AS ENUM ('NOT_STARTED', 'PASS', 'FAIL');
+
+-- CreateTable
+CREATE TABLE "redemptions" (
+    "id" SERIAL NOT NULL,
+    "created_at" TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updated_at" TIMESTAMP(6) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "user_id" INTEGER NOT NULL,
+    "kyc_attempts" INTEGER NOT NULL DEFAULT 0,
+    "kyc_status" "KycStatus" NOT NULL,
+    "jumio_account_id" VARCHAR,
+    "public_address" VARCHAR NOT NULL,
+    "email_verified" BOOLEAN NOT NULL DEFAULT false,
+    "dedupe_status" "DedupeStatus" NOT NULL,
```

Comment:
delete this, it's unrelated to this model

Reply:
I think we should have this live on the user